### ISSUE TITLE
Bug: Ratings and Reviews, Imports

### ISSUE THREAD

**Issue by nerdteacher** [mouse-reeve/bookwyrm#542]
I know I shouldn't feel bad, but I feel a little bad for constantly throwing things on here. I feel like I'm constantly complaining (when I know I'm not), but I just want to be helpful however I can. <3

So a few things I've noticed today. Sorry if you've already ran into them.

When I was trying to go through a couple books:
1. When writing a review, the stars (when using the review box) aren't working. Even after clicking any of them, they stayed blank. This also extends to the rating stars under the cover; I can click them, but they don't do anything (whereas before it would save the rating and reload the page).
Also, I might be mistaken, but this feels like it looks different from what it used to (but not intentionally different)?
![image](https://user-images.githubusercontent.com/77229429/105047142-3eb4c680-5a6a-11eb-949d-d40474fd1d69.png)

2. Also on the book review page, I can't edit 'reading activity' data. Both editing and deleting the reading activity doesn't work.

3. Trying to delete a review/post doesn't work. (When I click the three dots, nothing shows up.) I also can't respond to any posts (but I can boost/like them).

4. Importing using a CSV file returns a page saying "Server Error." But it still imports the books (I still get notifications saying my import is complete, and they appear in my library).

I think that's it. (Everything else on the book page still seems to work, btw. It's still possible to edit book details and switch editions; that all seems fine.)

---

Also, I just noticed that when you put in ISBN-13 that it automatically looks up the ISBN-10, and that is awesome! (Yes, I'm very easy to impress, but I like that you thought to implement that. It makes my former school librarian heart very happy.)

**Comment by mouse-reeve** [comment]
Do not feel bad! A lot of these sound like cache problems - can you do a hard refresh (command-shift-r on a mac) and try again? 

I'll take a look at the csv import situation, that sounds like something different.

**Comment by mouse-reeve** [comment]
import turned out to be a small fix, it was just a wrong link. Not sure why it gives a server error instead of a 404 though...

**Comment by rkingett** [comment]
I'd like to chime in here. The reviews work perfectly fine with the keyboard, which is a bummer because I thought HTML was supposed to just work with a mouse! I'm sorry I can't help here, just wanted to say it works on the keyboard side.

**Comment by mouse-reeve** [comment]
the stars one is odd to me! I'm glad it's working on keyboard. It's the only one of the things on that list I didn't change recently. If a hard refresh doesn't resolve it, can you let me know what browser and os you're using?

**Comment by rkingett** [comment]
Oh wait! Just tried the stars, not the ratings radio buttons. The stars do not work via the keyboard either but the ratings radio buttons in the review tab work.

**Comment by mouse-reeve** [comment]
when you say the stars, do you mean a different element in the same review form, or a different place in the application where there are stars? The star icons in the review form are labels, which should select the correct radio button when clicked. The radio button is visible to screen readers but not to visual users

**Comment by nerdteacher** [comment]
Okay, most everything is back to working for me! I can delete posts, and I can edit reading activities! I did do the hard refresh before you told me to, so whatever you did in the meantime fixed a lot of it.

The ratings and reviews are still a bit wonky, though. I'm using Windows/Vivaldi, for the record (but I can check with my partner, if that helps; he's got Linux/Vivaldi going). The one that's still remaining is this:

When I click the stars in this review space (or try to because they go back to being blank)...
![image](https://user-images.githubusercontent.com/77229429/105067415-f3a5ae00-5a7f-11eb-972e-c16a7f5b45fc.png)

... I noticed that they change here:
![image](https://user-images.githubusercontent.com/77229429/105067529-0f10b900-5a80-11eb-8cc0-3d68c0b9091e.png)

In completing the review bit, it doesn't save any ratings record. (So I can see under the book that I gave it a star rating, but it doesn't show up at the bottom or in my profile/feed.) If I try to write the review, the stars stay blank on the review post. (I hope that makes sense?)
